src/pudl/transform/eia.py

```diff
@@ -1278,6 +1274,9 @@ def harvested_entity(context, **clean_dfs):
         logger.info(f"Harvesting IDs & consistent static attributes for EIA {entity}")
         eia_settings = context.resources.dataset_settings.eia
         debug = context.op_config["debug"]
+        clean_dfs = {
+            df_name: PUDL_PKG.encode(clean_dfs[df_name]) for df_name in clean_dfs
+        }
```

Comment:
This ensures that every dataframe being fed into the harvesting process has been encoded -- including columns that aren't part of any schema.